content/pt/docs/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Documentação
+linkTitle: Docs
+menu: { main: { weight: 10 } }
+aliases: [/docs/workshop/*]
```

Comment:
I'm not sure how to handle this consistently, but we don't want to propagate legacy aliases like these. Maybe we should just mark them as legacy ... but it gets a bit complicated to handle all cases.

Anyhow, for now, let's just delete this unnecessary alias. WDYT @svrnm @theletterf et al.?

```suggestion
```

Reply:
yes, I think that's a good idea